client/components/AddSponsor.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { string } from 'prop-types';
+import axios from 'axios';
+
+const AddSponsor = ({ eventId, chapterId }) => {
```

Comment:
type this

Reply:
I have never used typescript, but have given it a go. I'm just not sure what type handlesubmit and handlechange are supposed to be?

Reply:
what editor are you using - do they have typescript plugins?

Reply:
@timmyichen I use vscode. I have the typescript hero extension but nothing else - any suggestions?